.github/workflows/pr-title-check.yml

```diff
@@ -0,0 +1,62 @@
+name: Check PR Title and Issue Consistency
+
+on:
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
+
+jobs:
```

Comment:
for GH, workflows are composed of jobs that run in order. This simple WF has a single job.